core/Plugin/Manager.php

```diff
@@ -765,15 +769,21 @@ public function getIncompatiblePlugins($piwikVersion)
      */
     public function getPluginsLoadedAndActivated()
     {
-        $plugins = $this->getLoadedPlugins();
-        $enabled = $this->getActivatedPlugins();
+        if (is_null($this->pluginsLoadedAndActivated)) {
```

Comment:
This method was called > 100k times because of `postEvent()`. I couldn't cache it in postEvent directly as we have no way to detect there whether a new plugin was loaded or unloaded etc. Therefore added a simple cache here.